source/linux-install.html.erb

```diff
@@ -0,0 +1,130 @@
+---
+title: Elixir Girls Guides Installing on Linux  
+---
+
+<%= partial "partials/guides-nav" %>
+
+<div id="wrapper">
+
+        <!-- Navigation -->
+        <%= partial "partials/guides-sidebar-nav" %>
+
+        <div id="page-wrapper">
+          <h1>Installing Elixir on Linux</h1>
+          <p>
+            In order to develop elixir applications, you will need to install elixir itself, erlang because elixir is made for the erlang virtual machine, nodejs for compiling assets in phoenix, and you'll most likely want a RDBMS (relational database management system) such as postgresql or MySQL. These can of course be installed by your distribution's package manager - in fact, there are official instructions for doing so for various distributions <a href="https://elixir-lang.org/install.html#unix-and-unix-like">here</a>.
+          </p>
+          <p>
+            You may, however, find yourself developing projects under different versions of elixir, erlang, and possibly even your RDBMS. For such situations, it's useful to use a version manager (those from a ruby background may be familiar with chruby, rbenv, or rvm). This software allows you to install multiple versions of your tools, and swith between them (mostly) automatically, using a file that specifies the version in each project.
+          </p>
+
+          <h2>The ASDF Version Manager</h2>
+          <p>
+            The <a href="https://github.com/asdf-vm/asdf">asdf version manager</a> allows version managerment of multiple different tools using a plugin system. Let's install asdf, then use it to install elixir, erlang, and postgresql.
```

Comment:
@catherine-jones I noticed that links were in a close-to-white colour, this might need a change, unless my middleman was being loopy.